pandas/tests/test_algos.py

```diff
@@ -579,7 +586,7 @@ def test_on_index_object(self):
 
         mindex = mindex.repeat(2)
 
-        result = pd.unique(mindex)
+        result = pd.unique(mindex).values
```

Comment:
Can you modify `expected` instead of modifying `result`.

Reply:
Done. Actually I think this test can be removed, since MultiIndex is covered by the `Index` fixture in the test above.

Reply:
Agreed - this can be removed.